### ISSUE TITLE
Error with Deploy package interface

### ISSUE THREAD

**Issue by noedelcroix** [fusioninventory/fusioninventory-for-glpi#3183]
Hi,
I have a problem with the package actions. When I click on the "+" to add a file or program, nothing happend... As you can see I have an error with the onclick event.
Thanks in advance for your help or the patch.
![Capture d’écran de 2021-06-03 17-23-15](https://user-images.githubusercontent.com/50598341/120670403-ae16da00-c490-11eb-93d8-ba980fa960be.png)
![Capture d’écran de 2021-06-03 17-23-53](https://user-images.githubusercontent.com/50598341/120670429-b2db8e00-c490-11eb-9f79-06fe0e5255f1.png)
![Capture d’écran de 2021-06-03 17-24-01](https://user-images.githubusercontent.com/50598341/120670447-b53de800-c490-11eb-9dfb-bded7a297866.png)

**Comment by ddurieux** [comment]
Version?

**Comment by noedelcroix** [comment]
Last version of GLPI and fusioninventory.
GLPI : 9.5.5
fusioninventory : 9.5+3.0

**Comment by ddurieux** [comment]
Do you have errors in the network tab of you browser debug tool ?

Do you have errors in the glpi log files (glpi/files/log/php-errors.log) ?

**Comment by noedelcroix** [comment]
No nothing. Just 200 status files and empty php-errors.log...

**Comment by ddurieux** [comment]
And run GLPI in debug?

**Comment by noedelcroix** [comment]
I think there isn't any errors. Isn't it a frontend error ?
![Capture d’écran de 2021-06-03 20-34-06](https://user-images.githubusercontent.com/50598341/120694991-1e325980-c4ab-11eb-82ef-6de784948a90.png)

**Comment by noedelcroix** [comment]
And deploy package look like to be loaded 
![Capture d’écran de 2021-06-03 20-36-00](https://user-images.githubusercontent.com/50598341/120695170-605b9b00-c4ab-11eb-8d67-2330ac995712.png)

**Comment by ddurieux** [comment]
the button not works always?

How to you have installed the plugin?

**Comment by noedelcroix** [comment]
Yes it's not working. I installed by downloading the last release on github and adding it into plugins folder of glpi.

**Comment by ddurieux** [comment]
do you have previous version of plugin, I mean in the folder glpi/plugins/?

If yes, do you have delete it before uncompress the new version?

**Comment by noedelcroix** [comment]
Just other plugins
![Capture d’écran de 2021-06-03 21-00-23](https://user-images.githubusercontent.com/50598341/120698045-ca297400-c4ae-11eb-8acd-b999b15bf106.png)

**Comment by ddurieux** [comment]
What is fox?

**Comment by noedelcroix** [comment]
It's a theme plugin (change color and logo). Even this plugin (fox) disabled, the problem is there

**Comment by ddurieux** [comment]
and disable too formcreator and mreporting?

**Comment by noedelcroix** [comment]
Sorry, I have the same problem... Maybe I will try to reinstall (an other time) fusioninventory plugin.
![Uploading Capture d’écran de 2021-06-03 21-05-20.png…]()

**Comment by noedelcroix** [comment]
![Capture d’écran de 2021-06-03 21-05-20](https://user-images.githubusercontent.com/50598341/120698628-9438bf80-c4af-11eb-8a7a-889376c3df07.png)

**Comment by ddurieux** [comment]
you have logout and login to test after disable these plugins?

**Comment by noedelcroix** [comment]
Yes of course. I just tried a new install and i unfortunately have the same problem...

**Comment by ddurieux** [comment]
I have no more idea ;(

**Comment by noedelcroix** [comment]
Ok thanks a lot for your help I will check and post the solution (if I will get it)

**Comment by ddurieux** [comment]
The best solution is to debug in the code

**Comment by noedelcroix** [comment]
yes, thanks

**Comment by noedelcroix** [comment]
I found the problem, so now, I have to solve it and to know why there is the problem.
The lib folder of fusioninventory plugin is not loaded and I don't know yet why (the folder exist and is accessible by his url).
![Capture d’écran de 2021-06-04 14-49-42](https://user-images.githubusercontent.com/50598341/120803972-68b3e480-c544-11eb-93db-40c02cb8b010.png)
![Capture d’écran de 2021-06-04 14-52-03](https://user-images.githubusercontent.com/50598341/120804012-74071000-c544-11eb-9fe9-1c794706bc2a.png)
